charts/elasticsearch/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+description: A helm chart for Elasticsearch
+home: https://github.com/wiremind/wiremind-helm-charts/tree/main/charts/kibana
+maintainers:
+  - name: Wiremind
+    url: https://github.com/wiremind/wiremind-helm-charts
+name: elasticsearch
+version: 8.5.1
+appVersion: 8.5.1
```

Comment:
test directly with the latest version:
- https://github.com/elastic/elasticsearch/releases

also bump the chart version